Source/WebCore/html/HTMLLinkElement.h

```diff
@@ -141,7 +141,7 @@ class HTMLLinkElement final : public HTMLElement, public CachedStyleSheetClient,
 
     String debugDescription() const final;
 
-    enum PendingSheetType : uint8_t { Unknown, ActiveSheet, InactiveSheet };
+    enum class PendingSheetType : uint8_t { Unknown, ActiveSheet, InactiveSheet };
```

Comment:
Active and Inactive would be better names. No need for the redundancy. r+ with that addressed.